### PR TITLE
Fallback to Okhttp instead

### PR DESCRIPTION
**Pull request by revonateB0T**
Resolve https://github.com/FooIbar/EhViewer/issues/1765
Resolve https://github.com/FooIbar/EhViewer/issues/1768
Resolve https://github.com/FooIbar/EhViewer/issues/1769

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -146,7 +146,7 @@ android {
                 "**.txt",
                 "**.bin",
                 "**.{html,mmd}", // Compose Destination
-                "**.properties", // Apache 5 HC version info
+                "okhttp3", // Okhttp public suffix
```
Comment (FooIbar) [inline app/build.gradle.kts:149]: Does not work, use /**.

**Review by FooIbar — APPROVED** [review]

File: app/build.gradle.kts (hunk 1 above)
Comment (revonateB0T) [reply app/build.gradle.kts:149]: I doube if we can remove it?
Seems okhttp internal use it to parse SET-Cookie header?

Comment (FooIbar) [reply app/build.gradle.kts:149]: Ain't cookies handled in the ktor layer?

Comment (revonateB0T) [reply app/build.gradle.kts:149]: > Ain't cookies handled in the ktor layer?

Yeah, you are right

Comment (revonateB0T) [reply app/build.gradle.kts]: Done

**Pull request merged**